plugins/convertPathData.js

```diff
@@ -951,6 +951,16 @@ function getIntersection(coords) {
   }
 }
 
+/**
+ * Does the same as `Number.prototype.toFixed` but without casting
+ * the return value to a string.
+ * @type {(num: number, precision: number) => number}
+ */
+function toFixed(num, precision) {
+  const pow = 10 ** precision;
+  return Math.round(num * pow) / pow;
```

Comment:
Hi @marvinhagemeister, I was reading your blog post and I noticed this potential issue:

`precision` can go up to 19 if I read this file right (but seems to default to 3?) so you can lose some... ah, er... precision when multiplying it with another large number:
```js
Math.round(Number.MAX_SAFE_INTEGER * 10**19) / 10**19  // off by one
Math.round(4294967295 * 10**19) / 10**19  // 4294967295.0000005
Math.round(16777215 * 10**19) / 10**19  // 16777214.999999998
```
I assume that's okay here because `num` won't be that big but I can't say for sure from context.

Reply:
Good catch, yeah with higher precision values this can become an issue. The `precision` setting is typically used to _reduce_ precision to make the SVG smaller. So far I've only seen precision settings of `2` or `3` in the wild.